packages/docs/docs/tutorial/get-started/create-and-integrate-the-first-app.mdx

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 2
+---
+
+import Columns from '@components/Columns';
+
+# Create and integrate the first application
+
+Click the "Create" button on the right, and the browser will redirect to the Application tab. Then click the "Create Application" button on this page.
+
+**Step 1**
+
+In the opening modal, choose which type of application you'd like to integrate:
+
+- **Native App** is an app that runs in a native environment, e.g., iOS app, Android app.
```

Comment:
nit: better align below? `, e.g.,` → `. E.g.,`

Reply:
ok with this